deps/install-non-oss.sh

```diff
@@ -39,3 +39,5 @@ mvn install:install-file -Dfile=vim25_65.jar        -DgroupId=com.cloud.com.vmwa
 
 # From https://my.vmware.com/group/vmware/details?downloadGroup=WEBCLIENTSDK67U2&productId=742
 mvn install:install-file -Dfile=vim25_67.jar        -DgroupId=com.cloud.com.vmware -DartifactId=vmware-vim25    -Dversion=6.7   -Dpackaging=jar
+
+mvn install:install-file -Dfile=pbm_65.jar -DgroupId=com.cloud.com.vmware -DartifactId=pbm -Dversion=6.5 -Dpackaging=jar
```

Comment:
is this really not in mvn central? I understood Vmware made all their libs redist??

Reply:
guess not, i now see the same for the old API. still seems off, though.

Reply:
can you add a link like in the one above?